test/format/t.c

```diff
@@ -375,19 +365,19 @@ static void
 usage(void)
 {
     fprintf(stderr,
-      "usage: %s [-1BlqRr] [-C wiredtiger-config]\n    "
-      "[-c config-file] [-h home] [name=value ...]\n",
+      "usage: %s [-1BqRt] [-C wiredtiger-config]\n    "
+      "[-c config-file] [-h home] [-T trace-options] [name=value ...]\n",
       progname);
     fprintf(stderr, "%s",
       "\t-1 run once then quit\n"
       "\t-B maintain 3.3 release log and configuration option compatibility\n"
       "\t-C specify wiredtiger_open configuration arguments\n"
       "\t-c read test program configuration from a file (default 'CONFIG')\n"
       "\t-h home directory (default 'RUNDIR')\n"
-      "\t-l log operations to a file\n"
+      "\t-L all|local\n"
```

Comment:
This `-T` and `-t` in the code, I think.

Reply:
Fixed in a698f4104.